Rlog/View/WorkSpace/WorkSpaceListView.swift

```diff
@@ -7,14 +7,92 @@
 
 import SwiftUI
 
+
 struct WorkSpaceListView: View {
+    
     var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
+        NavigationView {
+            ZStack {
+                Color.cardBackground
+                    .ignoresSafeArea()
+                
+                ScrollView {
+                    ForEach(models, id: \.self) { model in
+                        WorkSpaceCell(model: model)
+                    }
+                }
+                .padding(.top, 32)
+                .navigationBarTitleDisplayMode(.inline)
+                .toolbar {
+                    ToolbarItem(placement: .navigationBarLeading) {
+                        Text("근무지")
+                            .font(.title2)
+                            .fontWeight(.bold)
+                    }
+                    ToolbarItem(placement: .navigationBarTrailing) {
+                        Button(action: { print("button pressed") }){
+                            Image(systemName: "plus")
+                                .fontWeight(.bold)
+                        }
+                    }
+                }
+            }
+        }
     }
 }
 
-struct WorkSpaceListView_Previews: PreviewProvider {
-    static var previews: some View {
-        WorkSpaceListView()
+struct CustomModel: Hashable {
```

Comment:
추후에 CoreData의 모델로 변경되는 struct라면 TODO로 잊지않게 남겨두면 좋을 것 같아요 !
```suggestion
// TODO: CoreData의 Model로 변경
struct CustomModel: Hashable {
```

Reply:
주석작성을 해서 잊지 않도록 신경쓰겠습니다. 적용했습니다. 👍 [88202cb](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Bremen-DD/pull/8/commits/88202cb326bcef04c4f97808bebcf51c3e869402)